clang/test/Driver/baremetal-multilib-exclusive-group.yaml

```diff
@@ -0,0 +1,70 @@
+# REQUIRES: shell
```

Comment:
If `LIT_USE_INTERNAL_SHELL=1 llvm-lit ..` passes, you can remove `# REQUIRES: shell` (dependency on an external shell)